src/components/DifferentCards/AddressCard/AddressCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./AddressCard.css";
+const AddressCard = ({ address ,state,setState}) => {
+  return (
+    <div className="address-card__wrapper">
+      <p className="text-2 ">
+        <input type="radio" 
+        checked={address.name===state.name}
+        onChange={()=>{setState(()=>address)}}
+        /> {address.name}
+      </p>
+      <p className="">
```

Comment:
you can remove className if no class is applied.

Reply:
Thanks for noticing it